community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -121,7 +121,7 @@ public void doImport( Input input ) throws IOException
         NodeRelationshipLink nodeRelationshipLink = null;
         NodeLabelsCache nodeLabelsCache = null;
         long startTime = currentTimeMillis();
-        File badRelationshipsFile = new File( config.badFileName() );
+        File badRelationshipsFile = badRelationshipsFileName();
```

Comment:
The bad file might now be in a directory that does not exist. I think you should somehow deal with that. Either by throwing a nice exception, or by creating the missing parent directories. I'm mostly leaning to the former.